packages/ai-jsx/src/core/conversation.tsx

```diff
@@ -16,7 +17,7 @@ import _ from 'lodash';
  *    </ChatCompletion>
  * ```
  */
-export function SystemMessage({ children }: { children: Node }) {
+export function SystemMessage({ children }: { children: Node; metadata?: Record<string, Jsonifiable> }) {
```

Comment:
Noob TypeScript question: so you can define something in the type and not even use it? That's cool and now I realize I've needed this so much and never realized! 😅

Reply:
The key is that the identifiers on the left side of the `:` are _variable_ declarations -- the linter will get upset at you if you declare unused variables but nothing will get upset about unused fields on a type.